### ISSUE TITLE
Installation failed under MAC/OS

### ISSUE THREAD

**Issue by keimig** [IBM/deploy-ibm-cloud-private#16]
==> icp: TASK [addon : Adding label to proxy nodes] *************************************
==> icp: changed: [localhost] => (item=192.168.27.100)
==> icp: 
==> icp: TASK [addon : Adding label to management nodes] ********************************
==> icp: failed: [localhost] (item=192.168.27.111) => {"changed": true, "cmd": "kubectl label nodes 192.168.27.111 management=true --overwrite=true", "delta": "0:00:00.234490", "end": "2017-10-11 16:53:28.597171", "failed": true, "item": "192.168.27.111", "rc": 1, "start": "2017-10-11 16:53:28.362681", "stderr": "Error from server (NotFound): nodes \"192.168.27.111\" not found", "stderr_lines": ["Error from server (NotFound): nodes \"192.168.27.111\" not found"], "stdout": "", "stdout_lines": []}
==> icp: 
==> icp: PLAY RECAP *********************************************************************
==> icp: 192.168.27.100             : ok=195  changed=61   unreachable=0    failed=0   
==> icp: 192.168.27.101             : ok=0    changed=0    unreachable=1    failed=0   
==> icp: 192.168.27.102             : ok=0    changed=0    unreachable=1    failed=0   
==> icp: 192.168.27.111             : ok=0    changed=0    unreachable=1    failed=0   
==> icp: localhost                  : ok=109  changed=49   unreachable=0    failed=1   
==> icp: Playbook run took 0 days, 0 hours, 7 minutes, 10 seconds
==> icp: FATAL ERROR OCCURRED DURING INSTALLATION :-(
==> icp: 
==> icp: TASK [Checking Python interpreter] *********************************************
==> icp: changed: [192.168.27.100]
==> icp: fatal: [192.168.27.111] => Failed to connect to the host via ssh: ssh: connect to host 192.168.27.111 port 22: Host is unreachable
==> icp: fatal: [192.168.27.101] => Failed to connect to the host via ssh: ssh: connect to host 192.168.27.101 port 22: Host is unreachable
==> icp: fatal: [192.168.27.102] => Failed to connect to the host via ssh: ssh: connect to host 192.168.27.102 port 22: Host is unreachable
==> icp: 
==> icp: PLAY [Checking prerequisites] **************************************************

**Comment by tpouyer** [comment]
Can you tell me a bit more about your setup?

RAM Size:
HDD Size and Type:
CPU Speed # of Cores:
VirtualBox Version:
Vagrant Version:

Quick check, can you `vagrant ssh` into the VirtualBox vm and run `lxc list`?

**Comment by tobespc** [comment]
I have the same issue, lxc list for shows 

+--------------+---------+-----------------------+------+------------+-----------+
|     NAME     |  STATE  |         IPV4          | IPV6 |    TYPE    | SNAPSHOTS |
+--------------+---------+-----------------------+------+------------+-----------+
| cfc-manager1 | RUNNING | 192.168.56.101 (eth0) |      | PERSISTENT | 0         |
+--------------+---------+-----------------------+------+------------+-----------+
| cfc-worker1  | RUNNING | 192.168.56.102 (eth0) |      | PERSISTENT | 0         |
+--------------+---------+-----------------------+------+------------+-----------+
| cfc-worker2  | RUNNING | 192.168.56.103 (eth0) |      | PERSISTENT | 0         |
+--------------+---------+-----------------------+------+------------+-----------+

**Comment by tpouyer** [comment]
Interesting... the lxc instances should have more net devices than that.  The prereq software installation must be failing.  Can you grab the Cloud-init-output.log from the lxc instances so I can try to figure out why it’s failing.

lxc exec cfc-worker1 — cat /var/log/cloud-init-output.log

**Comment by rs2015** [comment]
I have failing for different reason.


MACBOOKPRO-20:deploy-ibm-cloud-private rpatel$ vagrant up
Bringing machine 'icp' up with 'virtualbox' provider...
==> icp: Importing base box 'bento/ubuntu-16.04'...
==> icp: Matching MAC address for NAT networking...
==> icp: Setting the name of the VM: IBM-Cloud-Private-dev-edition
==> icp: Clearing any previously set network interfaces...
==> icp: Preparing network interfaces based on configuration...
    icp: Adapter 1: nat
    icp: Adapter 2: hostonly
==> icp: Forwarding ports...
    icp: 22 (guest) => 2222 (host) (adapter 1)
==> icp: Running 'pre-boot' VM customizations...
A customization command failed:

["storageattach", :id, "--storagectl", "SATA Controller", "--port", 0, "--device", 0, "--type", "hdd", "--nonrotational", "on", "--medium", "/Users/rpatel/VirtualBox VMs/IBM-Cloud-Private-dev-edition/ubuntu-16.04-amd64-disk001.vmdk"]

The following error was experienced:

#<Vagrant::Errors::VBoxManageError: There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["storageattach", "87aa500c-97a3-4fbe-95e4-cd8fc76cf35b", "--storagectl", "SATA Controller", "--port", "0", "--device", "0", "--type", "hdd", "--nonrotational", "on", "--medium", "/Users/rpatel/VirtualBox VMs/IBM-Cloud-Private-dev-edition/ubuntu-16.04-amd64-disk001.vmdk"]

Stderr: VBoxManage: error: Could not find file for the medium '/Users/rpatel/VirtualBox VMs/IBM-Cloud-Private-dev-edition/ubuntu-16.04-amd64-disk001.vmdk' (VERR_FILE_NOT_FOUND)
VBoxManage: error: Details: code VBOX_E_FILE_ERROR (0x80bb0004), component MediumWrap, interface IMedium, callee nsISupports
VBoxManage: error: Context: "OpenMedium(Bstr(pszFilenameOrUuid).raw(), enmDevType, enmAccessMode, fForceNewUuidOnOpen, pMedium.asOutParam())" at line 179 of file VBoxManageDisk.cpp
VBoxManage: error: Invalid UUID or filename "/Users/rpatel/VirtualBox VMs/IBM-Cloud-Private-dev-edition/ubuntu-16.04-amd64-disk001.vmdk"
>

Please fix this customization and try again.

**Comment by tobespc** [comment]
that command returns a blank string
vagrant@master:~$ lxc list
+--------------+---------+-----------------------+------+------------+-----------+
|     NAME     |  STATE  |         IPV4          | IPV6 |    TYPE    | SNAPSHOTS |
+--------------+---------+-----------------------+------+------------+-----------+
| cfc-manager1 | RUNNING |                       |      | PERSISTENT | 0         |
+--------------+---------+-----------------------+------+------------+-----------+
| cfc-worker1  | RUNNING | 192.168.27.101 (eth0) |      | PERSISTENT | 0         |
+--------------+---------+-----------------------+------+------------+-----------+
| cfc-worker2  | RUNNING | 192.168.27.102 (eth0) |      | PERSISTENT | 0         |
+--------------+---------+-----------------------+------+------------+-----------+
vagrant@master:~$ lxc exec cfc-worker1 — cat /var/log/cloud-init-output.log
vagrant@master:~$ lxc exec cfc-worker2 — cat /var/log/cloud-init-output.log

**Comment by tpouyer** [comment]
it looks as if you need to change the 'base_segment' value in the `Vagrantfile` to something other than `192.168.27` like `192.168.56` and re-run:
```
vagrant destroy
vagrant up
```

**Comment by rs2015** [comment]
Yep, I already did. Install is successful. I logged in to ICP-UI.

Appreciate your help.
